packages/ui/src/components/va-time-picker/VaTimePickerColumn/VaTimePickerColumn.vue

```diff
@@ -49,26 +50,18 @@ export default defineComponent({
 
     watch(syncActiveItemIndex, (newVal) => { scrollTo(newVal) })
 
-    onMounted(() => scrollTo(syncActiveItemIndex.value, false))
-
-    const scrollTo = (index: number, animate = true) => {
-      nextTick(() => {
-        const children = rootElement.value!.children
-
-        const element = children[index] as HTMLElement
+    onMounted(() => {
+      if (rootElement.value) {
+        // 170 here it's a sum of gaps from "before" and "after" pseudo elements
```

Comment:
This will break large time pickers... Using something like useElementSize on fake divs can remove this constants.

Reply:
Or we could rely on offsetTop, no?

Reply:
> This will break large time pickers... Using something like useElementSize on fake divs can remove this constants.

Unfortunately pseudo elements can't be replaced with a divs, since it's may cause an error when user will scroll time-picker.